x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -192,8 +192,6 @@ func (k Keeper) ProcessTokenLogicModules(
 	logger = logger.With("actual_settlement_upokt", actualSettlementCoin)
 	logger.Info(fmt.Sprintf("About to start processing TLMs for (%d) compute units, equal to (%s) claimed", numClaimComputeUnits, actualSettlementCoin))
 
-	// TODO_MAINNET(@red-0ne): Add tests to ensure that a zero settlement coin
-	// due to integer division rounding is handled correctly.
```

Comment:
I'm assuming/confirming that this is remove because it's either already done or not a concern for some other reason?